code/controllers/subsystem/research.dm

```diff
@@ -20,33 +21,42 @@ SUBSYSTEM_DEF(research)
 
 	var/list/obj/machinery/rnd/server/servers = list()
 
-	var/list/techweb_nodes_starting = list()	//associative id = TRUE
-	var/list/techweb_categories = list()		//category name = list(node.id = TRUE)
-	var/list/techweb_boost_items = list()		//associative double-layer path = list(id = list(point_type = point_discount))
-	var/list/techweb_nodes_hidden = list()		//Node ids that should be hidden by default.
+	var/list/techweb_nodes_starting = list()		//associative id = TRUE

```

Comment:
these need to have autodoc comments

Reply:
Gonna go through that hell once I'm done.

Reply:
and that applies to all the vars u touched  😳